.github/actions/setup-env/action.yml

```diff
@@ -82,5 +88,5 @@ runs:
         ADMIN_PASSWD: 'Correct Horse Battery Staple'
       run: |
         USER_ID=$(id -u) GROUP_ID=$(id -g) \
-        docker compose -f docker-compose.yml up -d --build
+        docker compose -f docker-compose.yml up -d prestashop-git ${{ inputs.DATABASE }} --build
```

Comment:
Isn't the env variable enough to automatically build the proper DB service?

Reply:
I try it.